src/Riok.Mapperly.Abstractions/NamedMappingAttribute.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Diagnostics;
+
+namespace Riok.Mapperly.Abstractions;
+
+/// <summary>
+/// Specifies a custom name for the mapping. This can be used to reference in other mapping configurations.
+/// </summary>
+[AttributeUsage(AttributeTargets.Method)]
+[Conditional("MAPPERLY_ABSTRACTIONS_SCOPE_RUNTIME")]
+public sealed class NamedMappingAttribute(string name) : Attribute
```

Comment:
About documenting this. Should it be a separate section, or should it be part of the documentations where this can be used?

Reply:
In the end, I included the documentation for this, where it could be used. I didn't create a new section.